apertiumPoints.json

```diff
@@ -1,280 +1,695 @@
 {
-"type": "FeatureCollection",
```

Comment:
No need to format this one because whenever we use trimpairs.py to scrape the points, the format is going to revert back anyways.